CHANGELOG.md

```diff
@@ -4,6 +4,49 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.13.1] - 2023-03-28
+
+## 🚀 Features
+
+### Router homepage now supports redirecting to Apollo Studio Explorer ([PR #2282](https://github.com/apollographql/router/pull/2282))
+
+In order to replicate the landing-page experience (called "homepage" on the Router) which was available in Apollo Gateway, we've introduced a `graph_ref` option to the `homepage` configuration.  This allows users to be (optionally, as as sticky preference) _redirected_ from the Apollo Router homepage directly to the correct graph in Apollo Studio Explorer.
```

Comment:
I think this is broken at the moment and we should fix it. Currently I get a blank page if `graph_ref` isn't specified.

Reply:
Should be fixed via https://github.com/apollographql/router/pull/2870 which has been merged into this PR via https://github.com/apollographql/router/pull/2869/commits/54102d74e1ef72836236fa8ee4e52f653aefe6e4